ladybug/datacollection.py

```diff
@@ -883,7 +884,13 @@ def get_aligned_collection(self, value=0, data_type=None, unit=None):
         else:
             data_type = self.header.data_type
             unit = unit or self.header.unit
-        values = [value] * len(self._values)
+        if isinstance(value, Iterable) and not isinstance(value, (str, dict)):
```

Comment:
You should check this but checking for `str` is not going to cover `bytes` and specially in Python3 one can send a byte which is iterable and is not str or dict but is not what you want. This comment is * 3 for other two similar cases in this PR.

Reply:
Thanks for letting me know.  This affects more places than just the 3 in this PR, including a number of places from the last PR where you brought up a similar concern and recommended the use of `isinstance(value, Iterable)` without acknowledging `bytes`.  So I am guessing that the discovery of the `bytes` object as an iterable might be relatively new for both of us.

I am going to fix all places where this happens now.

Reply:
Done!